src/app/@theme/components/header/header.component.html

```diff
@@ -21,15 +21,7 @@
 <div class="header-container">
   <nb-actions size="small">
     <!-- Theme Switcher -->
-    <nb-action class="d-flex align-items-center">
```

Comment:
The theme switcher not being displayed could be because that the `<nb-action>` is to be under `<nb-actions>`. You could try to keep the `nb-action` part of header component and only move the internal tree to the theme switcher.

Reply:
This was the issue yeah. Fixed it now. I don't see the commits being reflected here. Will I need to make a new PR?